packages/vm/src/imports.rs

```diff
@@ -39,16 +39,12 @@ const MAX_LENGTH_QUERY_CHAIN_REQUEST: usize = 64 * KI;
 pub fn do_read<S: Storage, Q: Querier>(ctx: &mut Ctx, key_ptr: u32) -> VmResult<u32> {
     let key = read_region(ctx, key_ptr, MAX_LENGTH_DB_KEY)?;
 
-    let ffi_result = with_storage_from_context::<S, Q, _, _>(ctx, |store| Ok(store.get(&key)))?;
-    let value = match ffi_result {
-        Ok((value, gas_info)) => {
-            process_gas_info::<S, Q>(ctx, gas_info)?;
-            Ok(value)
-        }
-        Err((err, gas_info)) => {
-            process_gas_info::<S, Q>(ctx, gas_info)?;
-            Err(VmError::from(err))
-        }
+    let (result, gas_info) =
+        with_storage_from_context::<S, Q, _, _>(ctx, |store| Ok(store.get(&key)))?;
+    process_gas_info::<S, Q>(ctx, gas_info)?;
+    let value = match result {
+        Ok(value) => Ok(value),
+        Err(err) => Err(VmError::from(err)),
     }?;
```

Comment:
This is equivalent to:
```rust
let value = result?;
```

Reply:
Done in https://github.com/CosmWasm/cosmwasm/pull/489/commits/739faa1d641c4be1e60a4eda333900a7cc0718d6